src/main/java/com/github/alexthe666/iceandfire/client/render/tile/RenderDreadPortal.java

```diff
@@ -9,28 +9,28 @@
 import net.minecraft.util.ResourceLocation;
 import net.minecraft.util.math.BlockPos;
 import net.minecraft.world.EnumSkyBlock;
-import org.lwjgl.Sys;
 
+import javax.annotation.Nullable;
 import java.nio.FloatBuffer;
 import java.util.Random;
 
 public class RenderDreadPortal extends TileEntitySpecialRenderer<TileEntityDreadPortal> {
-    private static final ResourceLocation END_SKY_TEXTURE = new ResourceLocation("iceandfire:textures/environment/dread_portal_background.png");
-    private static final ResourceLocation END_PORTAL_TEXTURE = new ResourceLocation("iceandfire:textures/environment/dread_portal.png");
-    private static final Random RANDOM = new Random(31100L);
-    private static final FloatBuffer MODELVIEW = GLAllocation.createDirectFloatBuffer(16);
-    private static final FloatBuffer PROJECTION = GLAllocation.createDirectFloatBuffer(16);
-    private final FloatBuffer buffer = GLAllocation.createDirectFloatBuffer(16);
-
-    public void render(TileEntityDreadPortal te, double x, double y, double z, float partialTicks, int destroyStage, float alpha) {
+	private static final ResourceLocation END_SKY_TEXTURE = new ResourceLocation("iceandfire:textures/environment/dread_portal_background.png");
+	private static final ResourceLocation END_PORTAL_TEXTURE = new ResourceLocation("iceandfire:textures/environment/dread_portal.png");
+	private static final Random RANDOM = new Random(31100L);
+	private static final FloatBuffer MODELVIEW = GLAllocation.createDirectFloatBuffer(16);
+	private static final FloatBuffer PROJECTION = GLAllocation.createDirectFloatBuffer(16);
+	private final FloatBuffer buffer = GLAllocation.createDirectFloatBuffer(16);
+
+    public void render(@Nullable TileEntityDreadPortal te, double x, double y, double z, float partialTicks, int destroyStage, float alpha) {
```

Comment:
Nice- this seems to be the right annotation (for what this modpack has been using). Their using the javax annotations. This is a more explicit way to declare something can be nullable here.